### PR TITLE
Fix/msg update client

### PR DESCRIPTION
**Pull request by Vritra4**
parse header in MsgUpdateClient

### CODE REVIEW

**Comment by hanjukim** [comment]
Can we add some tests?

File: src/core/ibc/msgs/client/MsgUpdateClient.ts [hunk 1]
```diff
@@ -34,28 +35,37 @@ export class MsgUpdateClient extends JSONSerializable<
 
   public static fromData(data: MsgUpdateClient.Data): MsgUpdateClient {
     const { client_id, header, signer } = data;
-    return new MsgUpdateClient(client_id, header, signer);
+    console.log(JSON.stringify(header));
```
Comment (hanjukim) [inline src/core/ibc/msgs/client/MsgUpdateClient.ts:38]: remove console.log

**Review by hanjukim — CHANGES_REQUESTED** [review]

**Comment by Vritra4** [comment]
i'll add *.spec.ts also..

**Review by hanjukim — APPROVED** [review]

**Comment by hanjukim** [comment]
LGTM

**Pull request merged**